frontend/src/pages/log-in.ts

```diff
@@ -309,6 +333,16 @@ export class LogInPage extends LiteElement {
     `;
   }
 
+  async checkBackendInitialized() {
+    const resp = await fetch("/api/settings");
+    if (resp.status === 200) {
+      this.formStateService.send("BACKEND_INITIALIZED");
+    } else {
+      this.formStateService.send("BACKEND_NOT_INITIALIZED");
+      setTimeout(() => this.checkBackendInitialized(), 5000);
```

Comment:
To cover the edge case of the timer firing after navigating away from this view, we should store the timer ID and clear the timer when the component is disconnected in `disconnectedCallback`. It'll future proof a bit if we add links to the login page to app views that are accessible without logging in.

Here's an example in the `copy-button.ts` component: https://github.com/webrecorder/browsertrix-cloud/blob/main/frontend/src/components/copy-button.ts#L42